src/widgetastic/utils.py

```diff
@@ -686,3 +686,38 @@ def wrap(*args, **kwargs):
             raise StaleElementReferenceException("Couldn't handle it")
 
     return wrap
+
+
+class BasicFillViewStrategy(object):
+    """Used to fill view's widgets by default. It just calls fill for every passed widget
+
+    """
```

Comment:
Why an instance method over a classmethod, when there's no reliance on `self`?

Reply:
because it needs self in next Fill class.  I just want to make code similar